frontend/src/layout/navigation/SideBar/SideBar.tsx

```diff
@@ -204,7 +204,14 @@ function Pages(): JSX.Element {
                 <PageButton icon={<IconExperiment />} identifier={Scene.Experiments} to={urls.experiments()} />
             )}
             <LemonSpacer />
-            <PageButton icon={<IconGroupedEvents />} identifier={Scene.Events} to={urls.events()} />
+            {featureFlags[FEATURE_FLAGS.COLLABORATIONS_TAXONOMY] ? (
+                <>
+                    <PageButton icon={<IconGroupedEvents />} identifier={Scene.Events} to={urls.events()} />
+                    <PageButton icon={<IconGroupedEvents />} identifier={Scene.Taxonomy} to={urls.taxonomy()} />
```

Comment:
Feel free to tackle in a separate PR, but not sure "Taxonomy" is the right word from a user's perspective (is it clear enough?), also while the concept does encompass Actions, I'm not sure anyone looking for actions will think of looking here.

Reply:
I was struggling w naming this one. Maybe "Events & Actions Taxonomy/Definitions" but doesn't quite roll off the tongue. I'll keep this now and iterate on it later.